### PR TITLE
Implement API Endpoints, Schema and Seeding Scripts for Postgres

### PR DESCRIPTION
**Pull request by jorgencarlsen**

### CODE REVIEW

File: database/postgresql/generateDishes.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+};
+
+const numberOfRestaurants = 5000000;
+
+const createDishes = (numRestaurants) => {
+  const writer = csvWriter();
+  writer.pipe(fs.createWriteStream('./csv/dishes.csv'));
+  let dishCount = 0;
+  for (let i = 1; i < numRestaurants; i++) {
+    if (i % (Math.floor(numRestaurants / 10)) === 0) {
```
Comment (paprikatie) [inline database/postgresql/generateDishes.js:18]: Nice job implementing progress reports! Do we need this many?

File: database/postgresql/generateReviews.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+const fs = require('fs');
+const csvWriter = require('csv-write-stream');
+const faker = require('faker');
+
+const numberOfDishes = 10000064;
```
Comment (paprikatie) [inline database/postgresql/generateReviews.js:5]: such a specific number!

File: database/postgresql/schema.sql [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+CREATE TABLE IF NOT EXISTS reviews(
+  id SERIAL PRIMARY KEY,
+  dish_id INT NOT NULL REFERENCES dishes(id),
+  user_id INT NOT NULL REFERENCES users(id),
+  review_text VARCHAR(1000) NOT NULL,
+  dined_on VARCHAR(100),
+  rating FLOAT
+);
+
+\COPY restaurants (name, cuisine, phone, street, city, state, zip_code) FROM '/Users/jorgencarlsen/Hack-Reactor/sdc/popular-dishes-service/database/postgresql/csv/restaurants.csv' CSV HEADER;
```
Comment (paprikatie) [inline database/postgresql/schema.sql:42]: Nice!

**Review by paprikatie — APPROVED** [review]
Awesome progress nice job!
Have you run the seeding scripts and they work?
Good job using the router for server-side ~ next step to split into controllers/models?

File: database/postgresql/generateDishes.js (hunk 1 above)
Comment (jorgencarlsen) [reply database/postgresql/generateDishes.js:18]: Not sure, it updates about every 10% complete.

File: database/postgresql/generateReviews.js (hunk 2 above)
Comment (jorgencarlsen) [reply database/postgresql/generateReviews.js:5]: Haha, that's the number of dishes my last script generated.

File: database/cassandra/schema.cql [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  restaurant_name text,
+  username text,
+  user_id UUID,
+  dined_on date,
+  rating smallint,
+  review_text text,
+  PRIMARY KEY (dish_name, review_id, dined_on)
+)
+WITH CLUSTERING ORDER BY
+
```
Comment (Rdecoster) [inline database/cassandra/schema.cql:55]: Good job on schema

**Review by Rdecoster — COMMENTED** [review]
Great job on all of the work

**Pull request merged**